pkg/api/mlflow/service/artifact/storage/storage.go

```diff
@@ -46,6 +46,8 @@ func NewArtifactStorage(config *config.ServiceConfig) (Provider, error) {
 		switch u.Scheme {
 		case "s3":
 			return NewS3(config)
+		case "", "file":
+			return NewLocal(config)
 		}
```

Comment:
I think we should have a default case that returns an error

Reply:
ok